### PR TITLE
[Merged by Bors] - TY-2470 test requestNextFeedBatch

### PR DESCRIPTION
**Pull request by Robert-Steiner**
Ticket:
- [TY-2470]

[TY-2470]: https://xainag.atlassian.net/browse/TY-2470?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: discovery_engine/lib/src/api/events/engine_events.dart [hunk 1]
```diff
@@ -97,6 +99,7 @@ class EngineEvent with _$EngineEvent {
   @Implements<FeedEngineEvent>()
   const factory EngineEvent.nextFeedBatchRequestFailed(
     FeedFailureReason reason,
+    String? errors,
   ) = NextFeedBatchRequestFailed;
```
[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply discovery_engine/lib/src/api/events/engine_events.dart:103]: good point 👍

**Review by janpetschexain — APPROVED** [review]
lgtm

File: discovery_engine/test/integration/restore_feed_test.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import 'dart:io';
```
Comment (acrrd) [inline discovery_engine/test/integration/restore_feed_test.dart:15]: show

**Review by acrrd — APPROVED** [review]

**Comment by Robert-Steiner** [comment]
bors merge

**Pull request closed without merging**